doc/docker.md

```diff
@@ -33,9 +33,10 @@ The following are optional:
   of files downloaded from the server. This option is either `0` or `1`. The
   default is `0`.
 * `GEOIPUPDATE_VERBOSE` - Enable verbose mode. Prints out the steps that
-  `geoipupdate` takes. Set to **anything** (e.g., `1`) to enable.
-* `GEOIPUPDATE_CONF_FILE` - The path where the configuration file will be
-  written. The default is `/etc/GeoIP.conf`.
+  `geoipupdate` takes. Set to `1` to enable.
```

Comment:
Hmm, so requiring it be `1` will be a behaviour change. Perhaps we should make it work as originally described, unless we are going to do a major release. We should mention it in the change log if we're changing it.